src/buttons/FullscreenButton/FullscreenButton.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import OpenInFullIcon from '@mui/icons-material/OpenInFull';
+import CloseFullscreenIcon from '@mui/icons-material/CloseFullscreen';
+
+export const FullscreenButton = (props) => {
+  const { isFullscreen, setFullscreen } = props;
+
+  return (
+    <>
+      {isFullscreen ? (
+        <CloseFullscreenIcon onClick={setFullscreen} fontSize="inherit" />
+      ) : (
+        <OpenInFullIcon onClick={setFullscreen} fontSize="inherit" />
+      )}
+    </>
+  );
+};
+
+FullscreenButton.propTypes = {
+  /*
+   * Boolean to know if the element targetted is currently in fullscreen mode
+   */
+  isFullscreen: PropTypes.bool.isRequired,
+  setFullscreen: PropTypes.func.isRequired,
+};
```

Comment:
Same remarks as the `TableFilterRow` component about props documentation and props name :slightly_smiling_face: